src/Frontend/Core/Engine/Header.php

```diff
@@ -333,14 +341,21 @@ public function addOpenGraphData($key, $value, $overwrite = false)
         $this->addMetaData(array('property' => 'og:' . $key, 'content' => $value), $overwrite, 'property');
     }
 
+
     /**
      * Add Open Graph image
      *
      * @param string $image     The path to the image.
      * @param bool   $overwrite Should we overwrite the previous value?
+     * @param integer $width    The width of the image.
+     * @param integer $height   The height of the image.
      */
-    public function addOpenGraphImage($image, $overwrite = false)
+    public function addOpenGraphImage($image, $overwrite = false, $width = 0, $height = 0)
     {
+        // recast width and height
+        $width = (int) $width;
+        $height = (int) $height;
```

Comment:
Hmm, if you have both width and height, you add the meta data twice? Is this correct?